raspbian/applications/tvservice.md

```diff
@@ -0,0 +1,67 @@
+## TVSERVICE
+
+`tvservice` is a command line application used to get and set information about the display, targeted mainly at HDMI video and audio.
+
+Typing `tvservice` by itself will display a list of available command line options.
+
+### Options
+
+#### -p, --preferred
+
+Power on the HDMI output with preferred settings.
+
+#### -o, --off
+
+Powers off the display output. 
+
+**Important Note:** Powering off the output using this command will also destroy any framebuffers/dispmanx layers associated with the display. These are NOT re-established with a subsequent power on, so will result in a blank screen. 
+
+A better option is to use the [vcgencmd display_power](vcgencmd.md) option, as this will retain any framebuffers, so when the power is turned back on the display will be the returned to the previous power on state.
+
+#### -e, --explicit="Group Mode Drive"
+
+Power on the HDMI with the specified settings
+
+Group can be one of `CEA`, `DMT`, `CEA_3D_SBS`, `CEA_3D_TB`, `CEA_3D_FP`, `CEA_3D_FS`.  
+Mode is one of the modes returned from the -m, --mode option.  
+Drive can be one of `HMI`, `DVI`.  
```

Comment:
Is `HMI` correct or was this a typo of `HDMI` ?

Reply:
HDMI, fixed.